kamelet-support/src/main/java/io/kaoto/backend/api/service/deployment/generator/kamelet/KameletDeploymentGeneratorService.java

```diff
@@ -57,6 +59,18 @@ public String description() {
                 + "blocks or steps that can be reused on integrations.";
     }
 
+    @Override
+    public String validationSchema() {
+        try {
+            String schema = new String(CamelRouteFileProcessor.class
+                    .getResourceAsStream("kamelet-dsl.json").readAllBytes());
+            return schema;
+        } catch (IOException e) {
+            log.error(e.getMessage());
```

Comment:
If you only use one parameter in the `log.error` the trace is lost. Better to use the two parameter form of this function: `log.error("Your context/explanation", e);`.